app/Models/Feed.php

```diff
@@ -200,6 +200,13 @@ public function attributes($key = '') {
 		}
 	}
 
+	public function titleRewriteHandler() {
```

Comment:
```suggestion
	public function titleRewriteHandler(): FreshRSS_TitleRewriting_Handler {
```

Reply:
I can not do that since this method could return void. With PHP7.2, I can not type-hint the 2 values.